packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -453,3 +492,73 @@ export const Combobox = <T extends string | number>(props: ComboboxProps<T>) =>
     </div>
   );
 };
+
+// Returns the group item or the option item, depending on the row content type
+const ComboboxRow = <T extends string | number>({
+  item,
+  virtualRow,
+  getItemProps,
+  selectedItem,
+  highlightedIndex,
+  isLastItem,
+}: {
+  item: ComboboxOption<T> | ComboboxOption<T>['group'];
+  virtualRow: VirtualItem;
+  getItemProps: any;
+  selectedItem: ComboboxOption<T> | null;
+  highlightedIndex?: number;
+  isLastItem: boolean;
+}): JSX.Element => {
+  const styles = useStyles2(getComboboxStyles);
+
+  if (!item || (typeof item === 'string' && item === '') || (typeof item === 'object' && !item.value)) {
```

Comment:
Maybe use the functionality to check if it is a group option outside of this component?